VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -155,6 +155,11 @@ def log(self, message,level='Info'):
 
     def log_with_no_try_except(self, message, level='Info'):
         WriteLog = self.get_value_from_configfile('WriteLog')
```

Comment:
It will be cleaner to write wrappers around get_value_from_configfile()  with default-values as input and try-catch inside within them :
get_**str**_value_from_configfile(config, default_str_value)
get_**int**_value_from_configfile(config, default_int_value)